.eslintignore

```diff
@@ -0,0 +1,3 @@
+build/*.js
+config/*.js
+*.js
```

Comment:
`*.js` in ignore file 😮😮

I read this thread and understand why you did it
Solution with `--no-verify` sounds better but also is kostyl

https://mateacademy.slack.com/archives/C03QUGDSX5H/p1662711861278719